src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,49 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <>
+    <div
+      key={movie.imdbId}
+      className="card"
+      data-cy="Movie"
+    >
```

Comment:
```suggestion
    <div
      className="card"
      data-cy="Movie"
    >
```
There is no need for react fragment, as you already have a container.
Also, key attribute should be added inside an iteration, not here 
![image](https://github.com/mate-academy/react_movies-list-js/assets/96409563/328e594d-74f4-4242-9216-8f6f94e026f9)